api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -134,6 +136,34 @@
     return Audiences.COLLECTOR;
   }
 
+  /**
+   * Filters this audience.
+   *
+   * <p>The returned {@code Audience} may be the same, or a completely different one.</p>
+   *
+   * @param filter the filter
+   * @since 4.8.0
+   */
+  default @NonNull Audience filterAudience(final @NonNull Predicate<? super Audience> filter) {
+    if(filter.test(this)) {
```

Comment:
This could use the same inline if as the override in `ForwardingAudience.Single`. Alternatively, that method should use this format. Either way, it should probably be consistent.